include/bluetooth/hci.h

```diff
@@ -590,14 +594,20 @@ struct bt_hci_cp_le_set_random_address {
 	bt_addr_t bdaddr;
 } __packed;
 
-/* Advertising types */
-#define BT_LE_ADV_IND                           0x00
-#define BT_LE_ADV_DIRECT_IND                    0x01
-#define BT_LE_ADV_SCAN_IND                      0x02
-#define BT_LE_ADV_NONCONN_IND                   0x03
-#define BT_LE_ADV_DIRECT_IND_LOW_DUTY           0x04
-/* Needed in advertising reports when getting info about */
-#define BT_LE_ADV_SCAN_RSP                      0x04
+/* LE Advertising Types (LE Advertising Parameters Set)*/
+#define BT_LE_ADV_IND                  __DEPRECATED_MACRO 0x00
+#define BT_LE_ADV_DIRECT_IND           __DEPRECATED_MACRO 0x01
+#define BT_LE_ADV_SCAN_IND             __DEPRECATED_MACRO 0x02
+#define BT_LE_ADV_NONCONN_IND          __DEPRECATED_MACRO 0x03
+#define BT_LE_ADV_DIRECT_IND_LOW_DUTY  __DEPRECATED_MACRO 0x04
+/* LE Advertising PDU Types. */
+#define BT_LE_ADV_SCAN_RSP             __DEPRECATED_MACRO 0x04
```

Comment:
This is still needed for handling advertising reports from scan responses in the scan callback.

Reply:
Yes, to clarify I deprecated this defines in favour of new ones I put into gap.h.

The reason for this is that this is a combination of HCI input parameters and output parameters. I found that a bit confusing. The first four are not PDU types, but input to LE Set Advertising Parameters to instruct the controller how to advertise.
The last one is the PDU type for SCAN_RSP.


So here `BT_LE_ADV_DIRECT_IND_LOW_DUTY`  is the input parameter, and `BT_LE_ADV_SCAN_RSP` is the PDU type, with the same value.